data_structures/query_edge.hpp

```diff
@@ -50,7 +50,7 @@ struct QueryEdge
         }
         NodeID id : 31;
         bool shortcut : 1;
-        int distance : 30;
```

Comment:
You forgot to extend this: You need two fields here, one for `distance` and one for `duration`.